libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/ErrorResponse.java

```diff
@@ -82,33 +86,11 @@ public enum ErrorResponse {
             1069, "No mitigation journey route event found in cimit config"),
     UNSUPPORTED_MITIGATION_ROUTE(1070, "Unsupported mitigation route");
 
-    @JsonProperty("code")
     private final int code;
-
-    @JsonProperty("message")
     private final String message;
 
-    ErrorResponse(
-            @JsonProperty(required = true, value = "code") int code,
-            @JsonProperty(required = true, value = "message") String message) {
-        this.code = code;
-        this.message = message;
-    }
-
-    public int getCode() {
-        return code;
-    }
-
-    public String toJsonString() {
-        return String.format("{\"code\":%d,\"message\":\"%s\"}", code, message);
-    }
-
-    public String getMessage() {
-        return message;
-    }
-
     @JsonCreator
-    public static ErrorResponse forCode(int code) {
+    public static ErrorResponse forCode(@JsonProperty("code") int code) {
```

Comment:
This makes the serialisation and deserialisation for this class symmetric - previously it serialised to an object, be deserialised from an int (the annotation here turns this into a 'properties' JsonCreator, which reads the property from a JSON object).

Reply:
Having a standard list of error codes is potentially useful, but I agree we're not really making any use of it!

In principle, these errors should be monitored and visible in logs etc., with a corresponding run-book entry for each (i.e. what to do if each of the errors occurs).

(If you're not doing that, you may as well just inline the error messages...)